setup/security/authentication/oauth/cognito/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title:  "AWS Cognito"
+sidebar:
+  nav: setup
+---
+
+This page instructs you on how to obtain an OAuth 2.0 client ID and client secret for
+use with your AWS Cognito User Pools.
+
+## Setting up an AWS Cognito App Client
+
+1. Navigate to [https://aws.amazon.com/cognito/](https://aws.amazon.com/cognito/) and log in with your AWS credentials.
+2. Search for Cognito in the search bar.
+3. Select the user pools you want Spinnaker to use.
+4. At the side bar under "General settings", select "App clients", add a client.
+  - Make sure you select "Generate client secret."
+5. After that go to "App integration", then to "App client settings."
+  a) Select "Cognito User Pool" as one of the "Enabled Identity Providers."
```

Comment:
@joetancy I don't know if you've set yourself up to preview your changes, but these substeps might not resolve the way you want them to. If you _are_ previewing, and this looks ok, then ignore this comment. 

But Markdown usually wants sub steps to start with `1. `, just like the first-level steps. And the indentation causes Markdown to give them letters instead of numbers.

Reply:
apologies, fixed in the next commit!